ts/src/pro/binance.ts

```diff
@@ -1320,7 +1325,7 @@ export default class binance extends binanceRest {
         const message = {
             'id': messageHash,
             'method': 'account.status',
-            'params': this.signParams (this.extend (payload, params)),
+            'params': this.signParams (true, this.extend (payload, params)),
```

Comment:
if you only need this for the `fetchTrades` it is easier to do the opposite and make this true by default